views/owner.html

```diff
@@ -55,20 +55,24 @@ <h4>Recent events</h4>
       </div>
       <div class="col-sm-offset-2 col-sm-7">
         <div class="row">
-          <div class="col-sm-6 col-xs-12">
+          <div class="col-sm-4 col-xs-12">
             <h2 class="owner-page-title">Datasets <span class="badge" title="{{packages.summary.total}} published datasets">{{packages.summary.total}}</span></h2>
           </div>
-          <div class="col-sm-6">
+          <div class="col-sm-8 col-xs-6">
             {% if packages.results|length > 0 %}
```

Comment:
This if statement breaks UX if no results were found. You will be unable to search anything else.